templates/shift-scheduling/input.json

```diff
@@ -1,42 +1,88 @@
 {
-  "days": 5,
-  "workers": 5,
-  "preferences": {
-    "shift_type": [
-      {
-        "worker": 1,
-        "type": "morning"
-      },
-      {
-        "worker": 2,
-        "type": "day"
-      },
-      {
-        "worker": 3,
-        "type": "day"
-      },
-      {
-        "worker": 4,
-        "type": "night"
-      },
-      {
-        "worker": 5,
-        "type": "morning"
-      }
-    ]
-  },
-  "unavailable": [
-    {
-      "worker": 1,
-      "days": [1]
-    },
-    {
-      "worker": 4,
-      "days": [4, 5]
-    },
-    {
-      "worker": 5,
-      "days": [4, 5]
+  "workers": [
```

Comment:
This seems like a weird input to me. You have 5 workers, yet you require a minimum of 16 workers to cover the shifts, given the `required_workers` `count`.